uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1397,9 +1432,11 @@ sub make_jiv {
 =head3 make_nii($fileref, $data_dir)
 
 Creates NIfTI files associated with MINC files and append its path to the
-parameter_file table using the parameter_type "check_nii_filename".
+C<parameter_file> table using the C<parameter_type> C<check_nii_filename>.
 
-INPUTS: file hashref and data directory (typically C</data/project/data>)
+INPUTS:
+  - $fileref : file hashref
```

Comment:
hash ref to be consistent with your other changes earlier

Reply:
nice! Done.